caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
If you entered a "uchicago.edu" email. I'd redirect to Shibboleth, else it'll login normally? 

Would this solve the ITS issue that not all "uchicago.edu" emails have google app engine logins?